Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1518 @@
+
+    - The typical OpenStack director deployment scenario assumes that all the packages are installed on the overcloud-full image. The Overcloud qcow image (for example, overcloud-full.qcow2) needs to be patched with the following RPMs:
+
+        - nuage-bgp
+        - nuage-metadata-agent
+        - nuage-openstack-heat
+        - nuage-openstack-horizon
+        - nuage-openstack-neutron
+        - nuage-openstack-neutronclient
+        - nuage-openvswitch (Nuage VRS)
+        - nuage-puppet-modules-5.1.0
+        - selinux-policy-nuage
+        - nuage-topology-collector
+
+    - Uninstall Open vSwitch (OVS).
+    - Install VRS (nuage-openvswitch).
+
+    - Use nuage-puppet-modules-5.1.0.x86_64.rpm and the nuage_overcloud_full_patch.py script to patch to the Overcloud qcow image, uninstall Open vSwitch (OVS), and install VRS.
+
+    - For AVRS integration, the overcloud-full image is also patched with following 6WIND and Nuage AVRS RPMs:
+
+        - 6windgate-dpdk
+        - 6windgate-dpdk-pmd-mellanox-rdma-core
+        - 6windgate-dpdk-pmd-virtio-host
+        - 6windgate-fp
+        - 6windgate-fpn-sdk-dpdk
+        - 6windgate-fp-ovs
+        - 6windgate-linux-fp-sync
+        - 6windgate-linux-fp-sync-fptun
+        - 6windgate-linux-fp-sync-ovs
+        - 6windgate-linux-fp-sync-vrf
+        - 6windgate-product-base
+        - 6windgate-tools-common-libs-daemonctl
+        - 6windgate-tools-common-libs-libconsole
+        - 6windgate-tools-common-libs-pyroute2
+        - 6wind-openstack-extensions
+        - dkms
+        - nuage-metadata-agent (6wind version)
+        - nuage-openvswitch (6wind version)
+        - python-pyelftools
+        - selinux-policy-nuage-avrs
+        - virtual-accelerator-base
+
+
+* Adding Nuage Heat Templates ( `nuage-tripleo-heat-templates <../../nuage-tripleo-heat-templates>`_  )
+
+    - Nuage provides heat templates & environment files to configure neutron on controller and nuage-openvswitch & nuage-metadata-agent on compute nodes and we configure VA on computeavrs nodes.
+
+* Updating the TripleO Heat templates (also referred to as the puppet manifests)
+
+    - Some of the parameters in ``neutron.conf`` and ``nova.conf`` need to be configured in the Heat templates. The Nuage VRS and metadata agent also need to be configured. The values for these parameters depend on the Nuage VSP configuration.
+      We use ``neutron-nuage-config.yaml`` and ``nova-nuage-config.yaml`` environment files to configure these values.
+    - See the `Sample Templates`_ section for some probable values of the parameters in the ``neutron-nuage-config.yaml`` and ``nova-nuage-config.yaml`` files.
+    - For AVRS integration, some of the parameters in ``fast-path.env`` needs to be configured in the Heat templates. We use ``compute-avrs-environment.yaml`` environment file to configure these values.
```

Comment:
Revise the text:

Use ``compute-avrs-environment.yaml`` environment file to configure these values.